app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.order(:id)
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(name: params[:task][:name], description: params[:task][:description])
```

Comment:
You didn't make a strong params method in the controller, but you mentioned in the comprehension question answers exactly what strong params are! You can use them here.